SPECS/ocaml-augeas/ocaml-augeas.spec

```diff
@@ -5,24 +5,23 @@ ExcludeArch: %{ix86}
 Summary:        OCaml bindings for Augeas configuration API
 Name:           ocaml-%{srcname}
 Version:        0.6
-Release:        32%{?dist}
+Release:        33%{?dist}
 License:        LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://people.redhat.com/~rjones/augeas/files/
 Source0:        https://people.redhat.com/~rjones/%{srcname}/files/%{name}-%{version}.tar.gz
-Source1:        ocaml_files.py
-Source2:        macros.ocaml-rpm
  
 # Upstream patch to enable debuginfo.
 #Patch1:         0001-Use-ocamlopt-g-option.patch
 # Const-correctness fix for OCaml 4.09+
 #Patch2:         0002-caml_named_value-returns-const-value-pointer-in-OCam.patch
  
 BuildRequires:  make
-BuildRequires:  ocaml >= 3.09.0
+BuildRequires:  ocaml >= 5.1.1
 BuildRequires:  ocaml-findlib
 BuildRequires:  ocaml-ocamldoc
+BuildRequires:  ocaml-rpm-macros
```

Comment:
Why the new BR?

Reply:
I am not sure, the Fedora spec had it listed. I believe it is required for it to build successfully with ocaml `5.1.1` since it is getting a major jump

Reply:
Might be worth understanding this better. I think it's unlikely that packages would require RPM macros to function. And if they did, they should already have that dependency embedded inside them. One way or another, something seems unclear here.